force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -124,7 +147,13 @@ public with sharing class ClassesObjectsMethods {
     public static String findTheScientist() {
         // The list of scientists' names
         List<String> scientistNames = new List<String>{'Tim Berners-Lee', 'Alan Turing', 'Grace Hopper', 'Donald Knuth', 'Guido van Rossum', 'Ken Thompson', 'Stephen Hawking'};
-
+        String result = ''; 
+        for (String scientist: scientistNames) {
+            if (scientist.contains('g')) {
+                continue;
+            }
+            result = result + scientist + ',';
+        }
```

Comment:
Are you returning anything? Will this work if the letter is  G instead of g ? The continue is not what you want here.